packages/utils/src/utils/agent.utils.ts

```diff
@@ -34,6 +34,7 @@ export const createAgent = async ({
     identity,
     ...(host !== undefined && { host }),
     verifyQuerySignatures,
+    retryTimes: 10,
```

Comment:
Don’t we want to expose it as a parameter as well?

I would add `10` as a constant and explain why we chose 10.

Reply:
Good idea @lmuntaner ! Done.